crates/yakui-core/src/context.rs

```diff
@@ -8,7 +8,7 @@ use crate::dom::Dom;
 type Storage<T> = RefCell<Option<T>>;
 
 thread_local! {
-    static CURRENT_DOM: Storage<Dom> = RefCell::new(None);
+    static CURRENT_DOM: Storage<Dom> = const { RefCell::new(None) };
```

Comment:
y tho

Reply:
It's reasonably impactful actually!

```
This macro supports a special const {} syntax that can be used when the initialization expression can be evaluated as a constant. This can enable a more efficient thread local implementation that can avoid lazy initialization. For types that do not [need to be dropped](https://doc.rust-lang.org/std/mem/fn.needs_drop.html), this can enable an even more efficient implementation that does not need to track any additional state.
```
https://doc.rust-lang.org/std/macro.thread_local.html

Reply:
I believe it is unrelated to the recently stabilized `const` blocks and is only superficially similar

Reply:
Ooh, right!